04week/welcome-to-our-site/css/style.css

```diff
@@ -0,0 +1,102 @@
+body {
+    margin: 0;
+}
+
+@media (max-width: 992px) {
+
```

Comment:
Don't forget to change the container holding the 3 color bars and texts so they stack on top of each other.